output/elastic/outputelastic.go

```diff
@@ -105,8 +105,15 @@ func InitHandler(ctx context.Context, raw *config.ConfigRaw) (config.TypeOutputC
 	// map Printf to error level
 	logger := &errorLogger{logger: goglog.Logger}
 
+	// replace env var names with values on URL config
+	var resolvedURLs []string
+	for _, URL := range conf.URL {
```

Comment:
Sorry about my words, we should uppercase URL only if the variable name not starts with it, the variable `URL` should be `url` here.

Reply:
no worries. done